website/src/libs/api-v3/amazonPayments.js

```diff
@@ -70,24 +46,15 @@ api.authorize = (inputSet) => {
   });
 };
 
-api.closeOrderReference = (inputSet) => {
-  let amzPayment = connect(amazonPayments);
-  return new Promize((resolve, reject) => {
-    amzPayment.offAmazonPayments.closeOrderReference(inputSet, (err, response) => {
-      if (err) return reject(err);
-      return resolve(response);
-    });
-  });
-};
-
-api.executePayment = (inputSet) => {
```

Comment:
the same functionality is `closeOrderReference()` function. Further, I searched for executePayment and only found it for paypal:

```
vagrant@dhcppc6:~/projects/habitrpg/website$ grep -R executePayment .
./src/libs/api-v3/payments.js:api.paypalCheckoutSuccess = paypal.executePayment;
./src/libs/api-v3/payments.js~:api.paypalCheckoutSuccess = paypal.executePayment;
./src/controllers/top-level/payments/paypal.js~:exports.executePayment = function(req, res) {
./src/controllers/top-level/payments/paypal.js:exports.executePayment = function(req, res) {```